.github/workflows/deploy-gcp-tests.yml

```diff
@@ -901,9 +908,9 @@ jobs:
               --source-disk-zone=${{ vars.GCP_ZONE }} \
               --storage-location=us \
               --description="Created from commit ${{ env.GITHUB_SHA_SHORT }} with height ${{ env.SYNC_HEIGHT }} and database format ${{ env.DB_VERSION_SUMMARY }}" \
-              --labels="height=${{ env.SYNC_HEIGHT }},purpose=${{ inputs.disk_prefix }},commit=${{ env.GITHUB_SHA_SHORT }},state-version=${{ env.STATE_VERSION }},state-running-version=${RUNNING_DB_VERSION},initial-state-disk-version=${INITIAL_DISK_DB_VERSION},network=${NETWORK},target-height-kind=${{ inputs.disk_suffix }},update-flag=${UPDATE_SUFFIX},force-save=${{ inputs.force_save_to_disk }},updated-from-height=${ORIGINAL_HEIGHT},test-id=${{ inputs.test_id }},app-name=${{ inputs.app_name }}"
+              --labels="height=${{ env.SYNC_HEIGHT }},purpose=${{ inputs.disk_prefix }},commit=${{ env.GITHUB_SHA_SHORT }},state-version=${{ env.STATE_VERSION }},state-running-version=${RUNNING_DB_VERSION},initial-state-disk-version=${INITIAL_DISK_DB_VERSION},network=${NETWORK},target-height-kind=${{ inputs.disk_suffix }},update-flag=${UPDATE_SUFFIX},force-save=${{ inputs.force_save_to_disk }},updated-from-height=${ORIGINAL_HEIGHT},updated-from-disk=${ORIGINAL_DISK_NAME},test-id=${{ inputs.test_id }},app-name=${{ inputs.app_name }}"
```

Comment:
To test these changes, I need to run CI on this branch, and force it to save a disk with the new label:
https://github.com/ZcashFoundation/zebra/actions/runs/6179715964

(I had to make the branch name shorter due to bug #7559, but the code is the same.)

<img width="321" alt="Screenshot" src="https://github.com/ZcashFoundation/zebra/assets/8951843/297cde9f-b542-46e9-a13c-2c6dcf6de0dc">


#### Testing

The disks created by that action run must have an `updated-from-disk` label.
The "Get original cached state height from google cloud" step should log the disk name and height correctly.

Reply:
That run failed due to the long branch name bug, here is the same run on this branch name:
https://github.com/ZcashFoundation/zebra/actions/runs/6191030680

Reply:
> The "Get original cached state height from google cloud" step should log the disk name and height correctly.

This succeeded in the automatic CI run:

> 7560-merge-78dd443 Disk: 
> main Disk: zebrad-cache-main-3ec142d-v25-mainnet-tip-u-233519
> Description: Created from commit 3ec142d with height 2223799 and database format v25-1-1 changing to v25-2-0
> any branch Disk: zebrad-cache-7555-merge-6f8998d-v25-mainnet-tip-u-024427
> Description: Created from commit 6f8998d with height 2226239 and database format v25-1-1 changing to v25-2-0
> Selected Disk: zebrad-cache-7555-merge-6f8998d-v25-mainnet-tip-u-024427

https://github.com/ZcashFoundation/zebra/actions/runs/6180297352/job/16777443839?pr=7560#step:10:133

> zebrad-cache-7555-merge-6f8998d-v25-mainnet-tip-u-024427 height: 2226239

https://github.com/ZcashFoundation/zebra/actions/runs/6180297352/job/16777567561?pr=7560#step:14:84

But the tags didn't get tested because:
> Skipped cached state update because the new sync height 2226300 was less than 576 blocks above the original height 2226239 of zebrad-cache-7555-merge-6f8998d-v25-mainnet-tip-u-024427

Reply:
That worked!
https://github.com/ZcashFoundation/zebra/pull/7560#issuecomment-1722553396